src/doc/rustc/src/platform-support.md

```diff
@@ -158,6 +158,7 @@ target | std | host | notes
 `aarch64-unknown-freebsd` | ✓ | ✓ | ARM64 FreeBSD
 `aarch64-unknown-hermit` | ? |  |
 `aarch64-unknown-linux-gnu_ilp32` | ✓ | ✓ | ARM64 Linux (ILP32 ABI)
+`aarch64-unknown-linux-dynmusl` | ✓ | ✓ | ARM64 Linux (Musl with dynamic linking)
```

Comment:
I believe you will need to change bootstrap/dockerfiles somewhere under the `src/ci/docker` for builds of libstd/compiler to happen.

cc @Mark-Simulacrum  who knows the setup there better than me.

Reply:
I cleared the checkmarks for now, we can tackle that in a later MR.